packages/core/datadog-sourcemaps.gradle

```
@@ -189,6 +189,11 @@ private def runShellCommand(
     }
 }
 
-private static def getServiceName(variant) {
+private def getServiceName(variant) {
+    if (project.hasProperty("datadog")) {
```

won't something like this work? anyway, this doesn't really change anything.

```suggestion
    if (project.datadog != null && project.datadog.serviceName != null) {
```